fixed-hash/src/hash.rs

```diff
@@ -651,7 +651,7 @@ macro_rules! impl_arbitrary_for_fixed_hash {
 	( $name:ident ) => {};
 }
 
-// Implementation for enabled rand crate support.
+// Implementation for enabled `arbitrary` crate support.
```

Comment:
I know this is consistent with the rest of the comments but it's grammatically incorrect and hard to parse. Please consider using the previous suggestion (or rewrite it to your pleasing). It's fine if you correct only the additions in this PR.

Reply:
No worries, I've updated the two lines for arbitrary so that they are grammatically correct.